torch/distributed/optim/optimizer.py

```diff
@@ -128,8 +128,9 @@ def step(self, context_id):
 
         This will call :meth:`torch.optim.Optimizer.step` on each worker
         containing parameters to be optimized, and will block until all workers
-        return. The current distributed autograd
-        :class:`~torch.distributed.autograd.context` will be used globally.
+        return. The provided ``context_id`` will be used to retrieve the
```

Comment:
nit: Maybe slightly change the wording on this sentence? For example `The provided ``context_id`` will be used to retrieve the corresponding :class:~torch.distributed.autograd.context that contains the gradients to be applied on the parameters.`(or something similar?)